src/visualizers/panels/ForgeActionButton/Actions.js

```diff
@@ -158,6 +158,11 @@ define([
                 name: 'Upload artifact',
                 icon: 'swap_vert',
                 action: DeepForge.create.Artifact
+            },
+            {
+                name: 'Import storage paths',
```

Comment:
Something like "Import Existing Data" might be better than "Import storage paths"

Reply:
Renamed to `Import existing data`.